src/modules/roc_netio/target_libuv/roc_netio/udp_sender_port.cpp

```diff
@@ -142,8 +158,24 @@ void UdpSenderPort::write(const packet::PacketPtr& pp) {
             return;
         }
 
+        if (rate_limiter_.allow()) {
+            const double nb_ratio = nb_packet_counter_ != 0
+                ? (double)nb_packet_counter_ / packet_counter_
+                : 0.;
+            roc_log(LogDebug, "udp sender: total=%u non-blocking=%u nb_ratio=%.5f",
```

Comment:
Nitpick: `non-blocking` should be `non_blocking` or just `nb` (like a variable name). Just for consistency with the rest logging.